apps/docs/src/guide/types/numbers.md

```diff
@@ -0,0 +1,29 @@
+# Numbers
+
+In Sway, there are multiple types of numbers:
+
+1. `u8` (8-bit unsigned integer)
+2. `u16` (16-bit unsigned integer)
+3. `u32` (32-bit unsigned integer)
+4. `u64` (64-bit unsigned integer)
+5. `u256` (256-bit unsigned integer)
+
+Conveniently, all of these types are handled the same way in the TS SDK. This guide explains how to create and interact with Sway numbers while using the SDK.
+
+## Creating Numbers
+
+Just like Ethereum, when you pass in a number to a Sway program from JavaScript, you must first convert it to a `BigNumber` object. Here's how you can do that:
```

Comment:
```suggestion
Just like Ethereum, when you pass in a number to a Sway program from JavaScript, you must first convert it to a `BigNum` object. Here's how you can do that:
```

Reply:
Just ensuring that we are differentiating between [bn.js](https://github.com/indutny/bn.js/?tab=readme-ov-file) and [bignumber.js](https://mikemcl.github.io/bignumber.js/)